### PR TITLE
pt-BR: cr2 bare-metal translations

### PR DESCRIPTION
**Pull request by henrif75**
*Stats (with previous days merged):*
After: 3079 translated messages, 63 fuzzy translations, 318 untranslated messages.
#317 #1463

### CODE REVIEW

File: po/pt-BR.po [hunk 1]
```diff
@@ -13815,78 +13829,105 @@ msgid ""
 "If you don't have time to complete a certain part, don't worry: you can "
 "catch up in the next slot."
 msgstr ""
+"Esta parte do curso tem uma série de exercícios que se complementam. "
+"Estaremos fazendo-os ao longo do curso em vez de apenas no final. Se você "
```
Comment (azevedoalice) [inline po/pt-BR.po:13833]:
```suggestion
"Faremo-os ao longo do curso em vez de apenas no final. Se você "
```

File: po/pt-BR.po [hunk 2]
```diff
@@ -15680,128 +16321,158 @@ msgid ""
...
 "In this exercise, you're going to add a whole new Chromium feature, bringing "
 "together everything you already learned."
 msgstr ""
+"Neste exercício, você vai adicionar um novo recurso completo do Chromium, "
+"juntando tudo o que você já aprendeu."
 
 #: src/exercises/chromium/bringing-it-together.md:6
 msgid "The Brief from Product Management"
-msgstr ""
+msgstr "O Briefing da Gerência de Produto"
```
Comment (azevedoalice) [inline po/pt-BR.po:16398]:
```suggestion
msgstr "O Resumo da Gerência de Produto"
```

File: po/pt-BR.po [hunk 3]
```diff
@@ -18768,10 +20016,15 @@ msgid ""
 "the\n"
 "    /// interrupt is disabled.\n"
 msgstr ""
+"/// Seta ou limpa a máscara de interrupção.\n"
```
Comment (azevedoalice) [inline po/pt-BR.po:20019]:
```suggestion
"/// Define ou limpa a máscara de interrupção.\n"
```

**Review by azevedoalice — APPROVED** [review]
Left minor suggestions. LGTM.

**Comment by henrif75** [comment]
> Left minor suggestions. LGTM.

Thank you!

**Pull request merged**